plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -342,17 +328,9 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
               AnimationFinishStatus.ENDED -> it.onAnimatorEnding(type, this, owner)
             }
           }
-          // it could happen that some animators are cancelled inside onAnimatorStarting callback
-          // while high-level animation is starting and highLevelListener != null already,
-          // so we check on `started` flag to validate that high-level animation truly started
-          if (runningAnimatorsQueue.isEmpty() && highLevelAnimatorSet?.started == true) {
-            when (finishStatus) {
-              AnimationFinishStatus.CANCELED -> {
-                highLevelListener?.onAnimationCancel(animation)
-                highLevelListener?.onAnimationEnd(animation)
-              }
-              AnimationFinishStatus.ENDED -> highLevelListener?.onAnimationEnd(animation)
-            }
+          if (runningAnimatorsQueue.isEmpty()) {
```

Comment:
Could be collapsed with simialar `if` above?
Which is doing `mapTransformDelegate.setUserAnimationInProgress(false)`

Reply:
It will change the order for some callbacks, i'm not sure about it.